config/default.yml

```diff
@@ -105,6 +105,10 @@ blacklisted_external_identifiers:
     - P6578
     - P1004
 
+imported_from_properties:
```

Comment:
Naming suggestion (I know it's bikeshedding) but I would rather go with something like "blacklisted_reference_properties". Mostly because a) we might want to do similar things with other types of references b) it's more clear for a person who doesn't know how wikidata works or what "imported from" entails c) the community might want to have an "imported from" reference that are valid (importing from reliable sources instead of Wikimedia projects). That's why P143 has "from Wikimedia projects" in its title to make it clear it's going to be used for imports of unreliable source. (I don't know if I can explain it properly... got headache rn :()

Reply:
No, I think you're right, I'm always up for creating better variable names. I will change it to `ignored_reference_properties` as they are in fact ignored rather than blacklisted.